Contribute/index.md

```diff
@@ -47,6 +47,8 @@ Quick edits streamline the process to report and fix small errors and omissions
 4. When you're finished editing, scroll to the bottom of the page. In the **Propose changes** area, enter a title and optionally a description for your changes. The title will be the first line of the commit message. Select **Propose changes** to commit your changes:
```

Comment:
```suggestion
4. When you're finished editing, scroll to the bottom of the page. In the **Propose changes** area, enter a title and optionally a description for your changes. The title will be the first line of the commit message. Select **Propose changes** to create a new branch in your fork and commit your changes:
```

Reply:
Hi @jehchow! Yes, I think that would be fine. Would we need to update the picture if we do that?

Reply:
Hi @jehchow! Yes, I think that would be fine. Would we need to update the picture if we do that?